pkg/volume/volume.go

```diff
@@ -76,7 +76,12 @@ type Recycler interface {
 // Create adds a new resource in the storage provider and creates a PersistentVolume for the new resource.
 // Calls to Create should block until complete.
 type Creater interface {
-	Create() (*api.PersistentVolume, error)
+	// Provisions the resource in the underlying resource provider.
```

Comment:
Comment could be better with an example ("e.g. by allocating the underlying volume in a storage system")